wire-ios-sync-engine/Source/Data Model/Conversation+Join.swift

```diff
@@ -161,15 +169,20 @@ struct ConversationJoinRequestFactory {
 
     static let joinConversationsPath = "/conversations/join"
 
-    static func requestForJoinConversation(key: String, code: String) -> ZMTransportRequest? {
+    static func requestForJoinConversation(key: String, code: String, password: String? = nil) -> ZMTransportRequest? {
```

Comment:
From 2+ parameters I would prefer to format in new lines:
```suggestion
    static func requestForJoinConversation(
        key: String,
        code: String,
        password: String? = nil
    ) -> ZMTransportRequest? {
```

Reply:
Done: https://github.com/wireapp/wire-ios/commit/5b44e289b299f814b379510553914f61e3576624